README.md

```diff
@@ -47,6 +47,9 @@ If you are just looking for quickly spin up an instance of RH-SSO
 # Create Network Security Policy
 oc -n 6d70e7-tools process -f openshift/nsp.yaml -p 'NAMESPACE=6d70e7-tools' | oc -n 6d70e7-tools create -f -
```

Comment:
we probably want to remove or atleast highlight you WILL not need to run aporeto policies.